client/my-sites/plugins/controller.js

```diff
@@ -91,16 +92,17 @@ function renderPluginList( context, basePath ) {
 	lastPluginsQuerystring = context.querystring;
 	context.store.dispatch( setTitle( i18n.translate( 'Plugins', { textOnly: true } ) ) ); // FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
```

Comment:
Could we update the FIXME here? See: https://github.com/Automattic/wp-calypso/tree/master/client/components/data/document-head